students/ltrisolini/Lesson08/Circle_Test.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+import Circle
```

Comment:
module name which is equivalent to just a python file that ends in `*.py` should use all lower case naming; then you can:
`import circle` or `from circle import Circle`

it makes it clear if you're importing modules or classes